src/components/SignoffCardEntry/index.jsx

```diff
@@ -66,7 +66,7 @@ function getStatus(entry) {
 
 function SignoffCardEntry(props) {
   const classes = useStyles();
-  const { user, entry, name } = props;
+  const { user, entry, name, onSignoff, onRevoke } = props;
```

Comment:
Let's add these to `propTypes`.

Reply:
Note: If they are not required, then we should probably define a default value in `defaultProps`. Setting them to `null` should be good enough.

Reply:
Ah, I should actually make them required - my bad.